services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeMapper.kt

```diff
@@ -7,6 +7,7 @@ import fi.hel.haitaton.hanke.ContactType.TOTEUTTAJA
 import fi.hel.haitaton.hanke.domain.Hanke
 import fi.hel.haitaton.hanke.domain.HankeYhteystieto
 import fi.hel.haitaton.hanke.domain.SavedHankealue
+import fi.hel.haitaton.hanke.domain.resetFeatureProperties
```

Comment:
Actually, is this reset necessary here? If this is done when creating and updating hanke, there should be no need to reset them when transforming entity to domain.

Reply:
This line is no longer needed.

I think an equally valid question is: Do we need to save the features if we can just as well reset them on the way out?

Reply:
Hmm, I guess it does not make sense to save the properties at all.